sunset/code/modules/salvage/wreckage.dm

```diff
@@ -0,0 +1,100 @@
+
+#define SCRAP_METAL_STATS list(REC_PROB = 80, REC_MAXDROP = 5)
+#define SCRAP_TITANIUM_STATS list(REC_PROB = 40, REC_MAXDROP = 3)
+#define SCRAP_SILVER_STATS list(REC_PROB = 50, REC_MAXDROP = 2)
+#define SCRAP_GOLD_STATS list(REC_PROB = 60, REC_MAXDROP = 4)
+#define SCRAP_PLASMA_STATS list(REC_PROB = 50, REC_MAXDROP = 2)
+#define SCRAP_URANIUM_STATS list(REC_PROB = 40 , REC_MAXDROP = 1)
+#define SCRAP_BLUESPACE_STATS list(REC_PROB = 40, REC_MAXDROP = 1)
+#define SCRAP_METAL /obj/item/stack/ore/salvage/scrapmetal
```

Comment:
Yeah move these to another file in __DEFINE

Reply:
^